BUILD_FROM_SOURCE_README.md

```diff
@@ -0,0 +1,52 @@
+# Build Instructions
+
+The base tag this release is branched from is `20.0.1`
+
+---
+## Build
+Prerequisites: Building Keycloak from source requires the following:
+* JDK 11 (or newer)
+* Maven 3.5.4 (or newer).
+
+Building Keycloak server from source and creating an image, assumes the source files from the repository https://github.com/verrazzano/keycloak from branch oracle/release/20.0.1 or a branch created from oracle/release/20.0.1, is available under base directory _verrazzano_.
+
+To build only the Keycloak server, run:  
+```
+ cd verrazzano/keycloak/quarkus
+ # build the project for the first time to put required modules of Keycloak into local maven cache in package org.keycloak
+ mvn --settings ../maven-settings.xml -f ../pom.xml clean install -DskipTestsuite -DskipExamples -DskipTests
+ 
+ # build Keycloak Quarkus distribution
+ mvn --settings ../maven-settings.xml -f dist/pom.xml clean install
+```
+
+A successful maven build creates the distribution archive keycloak-20.0.1.tar.gz under keycloak/quarkus/dist/target.  
+
+Copy keycloak-<version>.tar.gz and other files required for building the image to verrazzano/keycloak-containers/server.  
+```
+ cd verrazzano/keycloak
+ cp quarkus/dist/target/keycloak-20.0.1.tar.gz keycloak/quarkus/container
```

Comment:
so it copies from `verrazzano/keycloak/quarkus/dist/target/keycloak-20.0.1.tar.gz` to `verrazzano/keycloak/keycloak/quarkus/container` ? some other path is mentioned in line above 
`verrazzano/keycloak-containers/server`

Reply:
Updated the instruction